README.md

```diff
@@ -43,9 +50,21 @@ Please visit the [doc folder](./doc) for more documentation about the Taxonomy E
 <img width="500" src="https://user-images.githubusercontent.com/25586296/194070730-302c0cac-800e-4c2a-80f6-67bad360bb74.png" />
 <img width="500" src="https://user-images.githubusercontent.com/25586296/194070769-99beed88-081f-435d-8fe0-b97e4c363fea.png" />
 <img width="500" src="https://user-images.githubusercontent.com/25586296/194070779-da4684d3-26ea-4fd9-8916-0e8c4fdea0c9.png" />
-
 </details>
 
+<h2> Weekly meetings </h2>
+- We e-meet Tuesdays at 17:00 Paris Time (16:00 London Time, 21:30 IST, 08:00 AM PT)
+- ![Google Meet](https://img.shields.io/badge/Google%20Meet-00897B?logo=google-meet&logoColor=white) Video call link: https://meet.google.com/xin-bwzm-xvj
+- Join by phone: https://tel.meet/nnw-qswu-hza?pin=2111028061202
+- Add the Event to your Calendar by [adding the Open Food Facts community calendar to your calendar](https://wiki.openfoodfacts.org/Events)
+- [Weekly Agenda](https://drive.google.com/open?id=1LL8-aiSF482xaJ1o0AKmhXB5QWfVE0_jzvYakq3VXys): please add the Agenda items as early as you can. Make sure to check the Agenda items in advance of the meeting, so that we have the most informed discussions possible. 
```

Comment:
```suggestion
- [Weekly Agenda](https://docs.google.com/document/d/1tdYkUmoRU8BxFPdCwtewoUi7PV8PmDlXtExOcPYyu-I): please add the Agenda items as early as you can. Make sure to check the Agenda items in advance of the meeting, so that we have the most informed discussions possible. 
```